script.js

```diff
@@ -19,4 +19,92 @@ document.addEventListener('DOMContentLoaded', () => {
+    const container03 = `<ul class="container1 containerx"><li><div class="works-pic"><img src="Assets/files/Snapshoot3.png" alt=""></div><div class="responsive-image1"><img src="Assets/files/Snapshoot desktop Portfolio3.jpg" alt="Responsive Image"></div></li><li><h2 class="works-heading tonic">Tonic</h2><h2 class="works-heading facebook">Facebook 360</h2><div class="canopy canopy1 canopyX"><img src="Assets/files/Frame 2.png" alt=""></div><div class="canopyX1 canopy-facebook"><div class="faceB">FACEBOOK</div><div class="circle"><i class="fa fa-circle"></i></div>Back End Dev<div class="circle"><i class="fa fa-circle"></i></div>2015</div></div><p class="works-paragraph deskpar1 deskparY">${works[i].mobiledescription}</p><p class="deskpar4">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p><ul class="languages languages-1"><li>html</li><li class="ruby">Ruby on rails</li><li>css</li><li>Javascript</li></ul><button class="project-3 project-33">See project</button></li></ul>`;
+    const container04 = `<ul class="container2 containery"><li><div class="works-pic"><img src="Assets/files/Snapshoot4.png" alt=""></div><div class="responsive-image2"><img src="Assets/files/Snapshoot desktop4.jpg" alt="Responsive Image"></div></li><li><h2 class="works-heading multi">${works[i].title}</h2><h2 class="works-heading uberheading">Uber Navigation</h2><div class="canopy canopy2 canopyX"><img src="Assets/files/Frame 2.png" alt=""></div><div class="canopyuber"><img src="Assets/files/Frame uber.jpg" alt=""></div><p class="works-paragraph deskpar2 deskparY">${works[i].mobiledescription}</p><p class="deskpar5">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p><ul class="languages languages-2 languages22"><li>html</li><li class="ruby">Ruby on rails</li><li>css</li><li>Javascript</li></ul><button class="project-4">See project</button></li></ul>`;
+    worksection.innerHTML = container01 + container02 + container03 + container04;
+  }
+  document.querySelector('.project-1').addEventListener('click', () => {
+    document.querySelector('.popup').classList.remove('popup-inactive');
+    document.querySelector('.popup').classList.add('popup');
+  });
+  document.querySelector('.project-2').addEventListener('click', () => {
```

Comment:
> When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- [ ] Looking at the video below, you can see that you have the same details and image on the pop-up for every project. that's not meant to be so. According to the requirement, the pop-up is generated dynamically depending on the project that was clicked. I'll suggest you also implement this. :+1: :smile: 

- - VIDEO REFERENCE:

https://github.com/denigit5/Portfolio/assets/107632780/b583be5c-0e1c-4c3a-b988-a399465747a4


- [ ] Also currently, clicking on the X close icon tends to refresh the page. this is not correct, the popup should close without the page being refreshed. I'll suggest you fix this. :wave: :smile: